scripts/run.e2e.sh

```diff
@@ -56,7 +56,17 @@ then
 	extra_build_args="-tags ledger_zemu"
 fi
 
-ACK_GINKGO_RC=true ginkgo build $extra_build_args ./tests/e2e
+ACK_GINKGO_RC=true ~/go/bin/ginkgo build $extra_build_args ./tests/e2e
+
+if [ "${COVERAGE_MODE:-}" == true ]
+then
+	echo 'Coverage mode enabled - re-creating coverage dir'
+	echo 'It requires the CLI binary to be built by build.sh with COVERAGE_MODE=true too'
+	export GOCOVERDIR=coverage
+	rm -rf ${GOCOVERDIR}
+	mkdir ${GOCOVERDIR}
```

Comment:
and mkdir -p

Reply:
good reminder - for e2e we may want to put it under `${GOCOVERDIR}/e2e` and ut coverage data to be under `${GOCOVERDIR}/ut`